CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+
+> **Note**
+> The new major protocol version must be precisely one greater than the current protocol version.
+> Any two consecutive epochs will therefore either have the same major protocol version, or the
+> later one will have a major protocol version that is one greater.
+
+> **Note**
+> There can be no duplicate committee members - each pair of credentials in a committee must be unique.
+
+Each governance action that is accepted on the chain will be assigned a unique identifier (a.k.a. the **governance action ID**),
+consisting of the transaction hash that created it and the index within the transaction body that points to it.
+
+#### Protocol Parameter groups
+
+We have grouped the protocol parameter changes by type,
+allowing different thresholds to be set for each group.
+We are not, however, restricting each protocol parameter governance action to be contained within one group.
+The maximum threshold of all the groups involved will apply to any given such governance action.
+
+The _network_,  _economic_ and _technical_ parameter groups collect existing protocol parameters that were introduced during the Shelley, Alonzo and Babbage eras.
+In addition, we introduce a new _governance_ group that is specific to the new governance parameters that will be introduced by CIP-1694.
+
+The **network group** consists of:
+* maximum block body size (`maxBBSize`)
+* maximum transaction size (`maxTxSize`)
+* maximum block header size (`maxBHSize`)
+* maximum size of a serialized asset value (`maxValSize`)
+* maximum script execution units in a single transaction (`maxTxExUnits`)
+* maximum script execution units in a single block (`maxBlockExUnits`)
+* maximum number of collateral inputs (`maxCollateralInputs`)
+
+The **economic group** consists of:
+* minimum fee coefficient (`minFeeA`)
+* minimum fee constant (`minFeeB`)
+* delegation key Lovelace deposit (`keyDeposit`)
+* pool registration Lovelace deposit (`poolDeposit`)
+* monetary expansion (`rho`)
+* treasury expansion (`tau`)
+* minimum fixed rewards cut for pools (`minPoolCost`)
+* minimum Lovelace deposit per byte of serialized UTxO (`coinsPerUTxOByte`)
+* prices of Plutus execution units (`prices`)
+
+The **technical group** consists of:
+* pool pledge influence (`a0`)
+* pool retirement maximum epoch (`eMax`)
+* desired number of pools (`nOpt`)
+* Plutus execution cost models (`costModels`)
+* proportion of collateral needed for scripts (`collateralPercentage`)
+
+The **governance group** consists of all the new protocol parameters that are introduced in this CIP:
+* governance voting thresholds ($P_1$, $P_{2a}$, $P_{2b}$, $P_3$, $P_4$, $P_{5a}$, $P_{5b}$, $P_{5c}$, $P_6$, $P_7$, $Q_1$, $Q_{2b}$, $Q_4$)
+* constitutional committee term limits
+* governance action expiration
+* governance action deposit (`govDeposit`)
+* DRep deposit amount (`drepDeposit`)
```

Comment:
If there will be a DRep deposits, should there be a reward address supplied in the DRep registration (or retirement?) certificate to receive the deposit amount back upon retirement?

Reply:
probably. There are still a few details we are ironing out regarding if the DReps will be retired instantly (and hence the deposit will just be a part of account balancing, like stake credential deposits) or if the funds have to be returned at the epoch boundary (thus requiring a reward account).